### DISCUSSION TITLE
Help Me! export word can't see in WPS

### DISCUSSION THREAD

**Discussion by LLLLLLLUO** [dolanmiu/docx#1643] [General]
If inscribe.  If Word opens and edits the first time, both WPS and Word can view and edit what follows, However, if WPS opens and edits the first time, only the title will remain, and the content will be lost

**Comment by dolanmiu** [comment]
Is this WPS? https://www.wps.com/

I have downloaded and installed it. Tried it with docx with no issues. It seems fine to me?

**Reply by zhangbaojia** [reply]
@dolanmiu  hi, This example is not compatible with WPS：
https://github.com/dolanmiu/docx/blob/master/demo/28-table-of-contents.ts
